source/SkiaSharpSource.Windows-netfx.slnf

```diff
@@ -21,6 +21,9 @@
       "SkiaSharp.Views.Uno\\SkiaSharp.Views.Uno.Reference\\SkiaSharp.Views.Uno.Reference.csproj",
       "SkiaSharp.Views.Uno\\SkiaSharp.Views.Uno.Skia\\SkiaSharp.Views.Uno.Skia.csproj",
       "SkiaSharp.Views.Uno\\SkiaSharp.Views.Uno.Wasm\\SkiaSharp.Views.Uno.Wasm.csproj",
+      "SkiaSharp.Views.Uno\\SkiaSharp.Views.Uno.WinUI.Mobile\\SkiaSharp.Views.Uno.WinUI.Mobile.csproj",
```

Comment:
Need to confirm if `SkiaSharp.Views.Uno.WinUI.Mobile.csproj` builds with netfx... I was having issues - but I think that was months ago so maybe it all works now :)